api/src/main/java/com/epam/pipeline/dao/user/UserDao.java

```diff
@@ -143,6 +148,32 @@ public PipelineUser updateUser(PipelineUser user) {
         return user;
     }
 
+    @Transactional(propagation = Propagation.MANDATORY)
+    public GroupStatus createGroupBlockingStatus(String groupName, boolean blockStatus) {
```

Comment:
Could you please add final to method arguments?
And double check in other places in current PR, please. (and local fields too)